client/src/main/java/org/apache/rocketmq/client/consumer/store/RemoteBrokerOffsetStore.java

```diff
@@ -213,6 +213,7 @@ public void updateConsumeOffsetToBroker(MessageQueue mq, long offset, boolean is
             requestHeader.setConsumerGroup(this.groupName);
             requestHeader.setQueueId(mq.getQueueId());
             requestHeader.setCommitOffset(offset);
+            requestHeader.setBname(mq.getBrokerName());
```

Comment:
Hi @drpmma ,

Why not just simply name it "brokerName" instead of "bname" to make it more straightforward? I felt confused for a second after I saw your description haha.

Reply:
The `bname` is not in the scope of this pr, see b6ff649291453d3dc2d702fbba1548d991b610ac for more information.

I guess `bname` is shorter than `brokerName` thus it is more efficient in tranport.

Reply:
@drpmma Oh, okay. That's reasonable.